src/main/java/com/minecolonies/coremod/entity/pathfinding/AbstractPathJob.java

```diff
@@ -951,7 +953,10 @@ protected boolean isPassable(@NotNull final IBlockState block)
         {
             if (block.getMaterial().blocksMovement())
             {
-                return block.getBlock() instanceof BlockDoor || block.getBlock() instanceof BlockFenceGate;
+                return block.getBlock() instanceof BlockDoor ||
+                         block.getBlock() instanceof BlockFenceGate ||
+                         block.getBlock() instanceof BlockConstructionTape ||
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') '||' should be on a new line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AOperator_Wrap)